### PR TITLE
Update data-lake-analytics.md

### PR DESCRIPTION
**Pull request by JeremyLikness**
Added overview, example code, and samples. Removed client section as there is only a management API. @CamSoper I will also have team review before sign-off.

### CODE REVIEW

**Comment by spboyer** [comment]
#sign-off

**Comment by openpublishbuild** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-1/api/overview/azure/index.yml) | :x:Error | | [Details](#user-content-api%2Foverview%2Fazure%2Findex.yml)
[]() | :x:Error | | [Details](#user-content-)

<a name="api%2Foverview%2Fazure%2Findex.yml"></a>
### [api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-1/api/overview/azure/index.yml)
 - **[Error]** Error transforming model &quot;W:\wvq0-o-unknown\raw-\zpc3wo0c.x5s\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

<a name=""></a>
### []()
 - **[Error]** Error transforming model &quot;W:\wvq0-o-unknown\raw-\zpc3wo0c.x5s\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201707181857040719-234%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by CamSoper** [comment]
@JeremyLikness - Two things:

1. Can you please add the strong types in the example?  Scott called out that he doesn't want vars in these examples.

2. To save me a step (and to make sure we preserve you as a contributor) can you please create a new PR with this same file going against the master branch instead?  I want to use the files from the aug-1-stage branch as the starting point (because I generated the templates for you guys there) but I want to check them directly into master.   I'll send an email to the team to that effect.

**Comment by openpublishbuild** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-1/api/overview/azure/index.yml) | :x:Error | | [Details](#user-content-api%2Foverview%2Fazure%2Findex.yml)
[]() | :x:Error | | [Details](#user-content-)

<a name="api%2Foverview%2Fazure%2Findex.yml"></a>
### [api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-1/api/overview/azure/index.yml)
 - **[Error]** Error transforming model &quot;W:\b5ch-o-14d97248\raw-\52wfmejc.2xi\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

<a name=""></a>
### []()
 - **[Error]** Error transforming model &quot;W:\b5ch-o-14d97248\raw-\52wfmejc.2xi\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201707181919187420-234%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api/overview/azure/data-lake-analytics.md [hunk 1]
```diff
@@ -65,19 +37,35 @@ dotnet add package Microsoft.Azure.Management.DataLake.Analytics
 
 ### Example
 
---Example overview--
+This example creates the clients to connect with and manage the analytics and store accounts.
 
 ```csharp
```
Comment (spboyer) [inline api/overview/azure/data-lake-analytics.md]: Make the code fluent, and inline the parameters. No vars

**Pull request closed without merging**